recipes/libtool/all/test_package/conanfile.py

```diff
@@ -1,80 +1,93 @@
-from conans import AutoToolsBuildEnvironment, CMake, ConanFile, tools
-from contextlib import contextmanager
-import glob
+from conan import ConanFile
+from conan.tools.build import cross_building, can_run
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.env import VirtualBuildEnv, VirtualRunEnv, Environment
```

Comment:
I'm not sure it's a good idea to mix up two different build systems, conan v2 doesn't seem to be robust to this case. I would create a test package for Autotools, and another one to test link to libltdl.